src/main/java/io/github/fusionflux/portalcubed/framework/model/PortalCubedModelLoadingPlugin.java

```diff
@@ -10,9 +12,38 @@
 public enum PortalCubedModelLoadingPlugin implements PreparableModelLoadingPlugin<EmissiveData> {
 	INSTANCE;
 
+	public static Selector currentSelectorBaking;
+
 	@Override
 	public void onInitializeModelLoader(EmissiveData emissiveData, Context ctx) {
 		ctx.modifyModelBeforeBake().register(ModelModifier.WRAP_PHASE, new MultiRenderTypeWrapper());
 		ctx.modifyModelAfterBake().register(ModelModifier.WRAP_PHASE, new EmissiveWrapper(emissiveData));
+
+		ctx.addModels(
+			PortalCubed.id("block/magnesium_fire_floor0"),
+			PortalCubed.id("block/magnesium_fire_floor1"),
+			PortalCubed.id("block/magnesium_fire_side_alt0"),
+			PortalCubed.id("block/magnesium_fire_side_alt1"),
+			PortalCubed.id("block/magnesium_fire_side0"),
+			PortalCubed.id("block/magnesium_fire_side1"),
+			PortalCubed.id("block/magnesium_fire_up_alt0"),
+			PortalCubed.id("block/magnesium_fire_up_alt1"),
+			PortalCubed.id("block/magnesium_fire_up0"),
+			PortalCubed.id("block/magnesium_fire_up1")
+		);
+		ctx.modifyModelAfterBake().register(ModelModifier.OVERRIDE_PHASE, (model, context) -> {
```

Comment:
this should be WRAP_PHASE

Reply:
I feel like override phase is more appropriate here but sure